dataflows/processors/dumpers/file_formats.py

```diff
@@ -203,3 +203,40 @@ def write_transformed_row(self, transformed_row):
 
     def finalize_file(self):
         self.writer.write(']')
+
+
+class GeoJSONFormat(JSONFormat):
+    def __init__(self, file, schema, **options):
+        writer = file
+        writer.write('{"type": "FeatureCollection","features":')
+        writer.__first = True
+        super(GeoJSONFormat, self).__init__(writer, schema, **options)
+
+    def write_row(self, row):
+        try:
+            geometry = {}
+            for k, v in row.items():
+                if self.fields[k].type == "geopoint":
+                    geometry = {"type": "Point",
+                                "coordinates" : self._FileFormat__transform_value(v, self.fields[k])}
+                    break
+                elif self.fields[k].type == "geojson":
+                    geometry = self._FileFormat__transform_value(v, self.fields[k])
+                    break
+        except Exception:
+            logging.exception('Missing point geometry in row %r', row)
+            raise
+        try:
+            properties = dict((k, self._FileFormat__transform_value(v, self.fields[k]))
```

Comment:
I'm pretty sure you can override `write_transformed_row` instead and avoid this bit of code.